docs/how-to-guides/openstack-swift.md

```diff
@@ -0,0 +1,22 @@
+Sometimes it is desirable to cache swift storage objects inside the charm for faster processing and reducing the number of requests to the swift server.
+
+This guide will demonstrate how to deploy this charm along with Openstack swift storage.
+
+First, connect to your openstack instance with your credentials, sourcing an .rc file:
+```
+source myfile.rc
+```
+Then list all the objects on the openstack containers in debug mode:
+```
+openstack object list default -vv
+```
+You will see a debug log as output. The URL should be located in the URL section of a curl command. The swift URL will look like this:
```

Comment:
```
openstack object list default -vv 2>&1 | grep "^REQ: curl" | grep AUTH | cut -d'"' -f2 | cut -d'?' -f1
```
Not sure that it's reliable but maybe worth mentioning as example.

Reply:
Adding it as another option :+1: